### PR TITLE
[ebay.ca] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/issues/13603

### CODE REVIEW

File: src/chrome/content/rules/ebay.ca.xml [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+	For other eBay coverage, see ebay.com.xml	
+	
+	
+	Non-functional subdomains:
+	
+		- ebay.ca	(h)
+		- arribada	(t)
+		- buyvrfn	(m)
+		- cafr	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.ca.xml]: WFM

**Comment by youdly** [comment]
Added `cafr` subdomain. WFM too.

**Comment by J0WI** [comment]
Looks like we need to redirect them to `www`, just like `^`.

**Comment by youdly** [comment]
Added redirect

**Pull request merged**